packages/lesswrong/unitTests/sql/SelectFragmentQuery.tests.ts

```diff
@@ -0,0 +1,57 @@
+import SelectFragmentQuery from "../../lib/sql/SelectFragmentQuery";
+import { runTestCases } from "../../lib/sql/tests/testHelpers";
+
+describe("SelectFragmentQuery", () => {
```

Comment:
Good point! I'm actually going to comment that function out for now as it's currently unused - I won't delete it quite yet, just in case we need to reseurrect it in the near future.